IHP/ErrorController.hs

```diff
@@ -143,6 +144,51 @@ patternMatchFailureHandler exception controller additionalInfo = do
             respond $ responseBuilder status500 [(hContentType, "text/html")] (Blaze.renderHtmlBuilder (renderError title errorMessage))
         Nothing -> Nothing
 
+-- Handler for 'IHP.Controller.Param.ParamNotFoundException'
+paramNotFoundExceptionHandler :: (Show controller, ?requestContext :: RequestContext) => SomeException -> controller -> Text -> Maybe (IO ResponseReceived)
+paramNotFoundExceptionHandler exception controller additionalInfo = do
+    case fromException exception of
+        Just (exception :: Param.ParamNotFoundException) -> Just do
+            let (Param.ParamNotFoundException paramName) = exception
```

Comment:
Does this work:
```suggestion
        Just (exception@(Param.ParamNotFoundException paramName)) -> Just do
```
or is this a quirk with `fromException`?

Reply:
Good point, just changed it 👍